### ISSUE TITLE
Change in Yandex Zen (avatars.mds.yandex.net)

### ISSUE THREAD

**Issue by aslian** [qsniyg/maxurl#1070]
Links like
https://avatars.mds.yandex.net/get-zen_doc/135437/pub_5d02297793658100b1ada4e0_5d022fd9778de800b0d69901/scale_600
https://avatars.mds.yandex.net/get-zen_doc/46847/pub_5accfec879885ebb4ce18e76_5accff227ddde8b6903295d8/scale_1200
need to be rewritten to
https://avatars.dzeninfra.ru/get-zen_doc/135437/pub_5d02297793658100b1ada4e0_5d022fd9778de800b0d69901/orig
https://avatars.dzeninfra.ru/get-zen_doc/46847/pub_5accfec879885ebb4ce18e76_5accff227ddde8b6903295d8/orig

Basically just avatars.mds.yandex.net -> avatars.dzeninfra.ru.
